versions.txt

```diff
@@ -481,10 +481,8 @@ grpc-google-cloud-shell-v1:2.48.0:2.49.0-SNAPSHOT
 proto-google-cloud-shell-v1:2.48.0:2.49.0-SNAPSHOT
 google-cloud-speech:4.44.0:4.45.0-SNAPSHOT
 grpc-google-cloud-speech-v1:4.44.0:4.45.0-SNAPSHOT
-grpc-google-cloud-speech-v1beta1:2.44.0:2.45.0-SNAPSHOT
 grpc-google-cloud-speech-v1p1beta1:2.44.0:2.45.0-SNAPSHOT
 proto-google-cloud-speech-v1:4.44.0:4.45.0-SNAPSHOT
-proto-google-cloud-speech-v1beta1:2.44.0:2.45.0-SNAPSHOT
```

Comment:
qq, why are these two removed?

Reply:
For speech, the whole v1beta1 version does not exist anymore, see [googleapis](https://github.com/googleapis/googleapis/tree/master/google/cloud/speech)

Reply:
I can't really tell from the diffs on Github, but I'm guessing the entire subfolder is removed in this PR?

Reply:
Actually I think still see the folders in the branch. Can we just remove the folders in google-cloud-java?

Reply:
I think they should be removed already. Let's merge this in first, I'll create a follow up cleanup PR if the folders still exist.